### ISSUE TITLE
NX-OS: Parsing error missing some config in peer templates

### ISSUE THREAD

**Issue by ratulm** [batfish/batfish#5081]
From https://networktocode.slack.com/archives/CCE02JK7T/p1572345429063000
```
router bgp 64603
...
  template peer peer-rr-overlay
    local-as 64603
    address-family l2vpn evpn
      send-community
      send-community extended
      route-map rm_rr_overlay_out out
  neighbor 10.0.0.1
    inherit peer peer-rr-overlay
    remote-as 64602
    no shutdown
    update-source Ethernet1/10
```
User report: the config template is not supported thus the bgp session is down.

**Comment by dhalperi** [comment]
This seems in my wheelhouse

**Comment by sdktr** [comment]
A nxos bgp template is a ‘global’ bgp property, which can be inherited to a vrf as well

**Comment by dhalperi** [comment]
@sdktr - we have BGP template support entirely, we must just be making some parsing error in the middle that is losing one property and causing issues.

**Comment by dhalperi** [comment]
So I added tests, and everything here was parsed and inherited correctly. I added a minor conversion fix in #5087 to handle this incomplete test config. I think the original issue on slack has to do with the `...`; I'm following up there.
